infrastructure/terraform/main.tf

```diff
@@ -45,45 +58,59 @@ module "ecs-cluster" {
+  ec2_instance_type          = var.ec2_instance_type
+  ec2_image_id               = var.ec2_image_id
+  asg_max_instance_count     = var.asg_max_instance_count
+  asg_min_instance_count     = var.asg_min_instance_count
+  asg_desired_instance_count = var.asg_desired_instance_count
+  application_subnet_ids     = data.terraform_remote_state.networks.outputs.application_ids
 }
 
 module "secrets" {
   source = "./module-secrets"
 
   stack_name  = local.stack_name
   name_prefix = local.name_prefix
-  environment = "${var.environment}"
-  kms_key_id  = "${data.terraform_remote_state.regional-infrastructure.outputs.poc_ecs_cluster_kms_key_id}"
-  secrets     = "${var.secrets}"
+  environment = var.environment
+  kms_key_id  = data.terraform_remote_state.services-stack-configs.outputs.services_stack_configs_kms_key_id
+  secrets     = data.vault_generic_secret.secrets
 }
 
 module "ecs-services" {
   source = "./module-ecs-services"
 
   stack_name  = local.stack_name
   name_prefix = local.name_prefix
-  environment = "${var.environment}"
-
-  vpc_id                  = "${lookup(var.vpc_id, var.aws_region)}"
-  aws_region              = "${var.aws_region}"
-  ssl_certificate_id      = "${var.ssl_certificate_id}"
-  zone_id                 = "${var.zone_id}"
-  zone_name               = "${var.zone_name}"
-  internal_cidrs          = "${var.internal_cidrs}"
-  application_ids         = "${data.terraform_remote_state.networks.outputs.application_ids}"
-  application_cidrs       = "${data.terraform_remote_state.networks.outputs.application_cidrs}"
-  ecs_cluster_id          = "${module.ecs-cluster.ecs_cluster_id}"
-  task_execution_role_arn = "${module.ecs-cluster.ecs_task_execution_role_arn}"
-
-  secrets_arn_map = "${module.secrets.secrets_arn_map}"
+  environment = var.environment
+
+  vpc_id                          = data.terraform_remote_state.networks.outputs.vpc_id
+  aws_region                      = var.aws_region
+  ssl_certificate_id              = var.ssl_certificate_id
+  zone_id                         = var.zone_id
+  external_top_level_domain       = var.external_top_level_domain
+  internal_top_level_domain       = var.internal_top_level_domain
+  internal_cidrs                  = var.internal_cidrs
+  mgmt-eu-west-1_cidrs            = var.mgmt-eu-west-1_cidrs
+  application_ids                 = data.terraform_remote_state.networks.outputs.application_ids
+  application_cidrs               = data.terraform_remote_state.networks.outputs.application_cidrs
+  ecs_cluster_id                  = module.ecs-cluster.ecs_cluster_id
+  task_execution_role_arn         = module.ecs-cluster.ecs_task_execution_role_arn
+  docker_registry                 = var.docker_registry
+  app_version_test_data_generator = var.app_version_test_data_generator
+  docker_container_port           = var.docker_container_port
+
+  log_level       = var.log_level
+  secrets_arn_map = module.secrets.secrets_arn_map
 }
+
+# output "var_secrets" {
```

Comment:
These should come out

Reply:
Ah yes. Those had been there for testing / debugging. Now removed.